### DISCUSSION TITLE
NOT moves items from different Organizations

### DISCUSSION THREAD

**Discussion by msimonyants** [dani-garcia/vaultwarden#2695] [Q&A]
Hello world))
please help us with this problem. We have organization number1 and several items there. We are creating another organization number2. Next, we want to move items from Organization number1 to Organization number2 - but that's not possible. Why? I only found this on the [official page](https://bitwarden.com/help/sharing/)
It turns out that now this functionality is not available on the local version of the server? Do I need to use only the cloud version?

**Comment by jjlin** [comment]
AFAIK, currently the best you can do is clone the item from the org1 vault to your personal vault, and then move the cloned entry to the org2 vault. You need to have admin or higher privileges in org1 to do this. In principle, you should be able to clone directly from the org1 vault to the org2 vault, but there seems to be some bug or limitation that doesn't allow selecting collections for the destination vault.

In general, Bitwarden currently doesn't have very good support for operating on org entries. If that's important to you, consider voting on relevant feature requests, like

* https://community.bitwarden.com/t/add-unshare-option-1-click-move-organization-vault-item-to-individual-vault/604
* https://community.bitwarden.com/t/ability-to-move-items-between-organizations/8170

**Comment by msimonyants** [comment]
@jjlin thank u!! it works! and i added a voice to support feature of direct transfer between organizations
